services/telemetry/telemetry.go

```diff
@@ -256,7 +257,7 @@ func (s *Service) sendV1Request(ctx context.Context, data []byte) error {
 	return nil
 }
 
-func (s *Service) makeV2Payload(serverUUID string) (*reporter.ReportRequest, error) {
+func (s *Service) makeV2Payload(serverUUID string, sttEnabled, iaEnabled bool) (*reporter.ReportRequest, error) {
```

Comment:
maybe wrap it up to struct?

Reply:
@palash25 Seems like David's comment was not resolved...

Reply:
i will push it before merging

Reply:
@palash25 `push -> re-review -> merge`, issues should be addressed and approved by commenter

Reply:
And it seems that this function can accept just `settings`

Reply:
done